elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpersUtil.java

```diff
@@ -0,0 +1,217 @@
+    public static final String SCHEMA_TYPE_SECURITY = "security";
+    public static final String SCHEMA_TYPE_VARIABLE = "variable";
+    public static final String INVALID_ERROR_MSG = "Incompatible or invalid config";
+
+    private static final String TABLE_CONFIG_PATH = "tables/";
+    private static final String SECURITY_CONFIG_PATH = "security.hjson";
+    private static final String VARIABLE_CONFIG_PATH = "variables.hjson";
+
+    private static final String ELIDE_TABLE_VALIDATION_SCHEMA = "elideTableSchema.json";
+    private static final String ELIDE_SECURITY_SCHEMA = "elideSecuritySchema.json";
+    private static final String ELIDE_VARIABLE_SCHEMA = "elideVariableSchema.json";
+
+    private static final String CHAR_SET = "UTF-8";
+    private static final String NEW_LINE = "\n";
+
+    /**
+     * Converts hjson input string to valid json string.
+     * @param hjson : input config in hjson format
+     * @return valid json string
+     */
+    public String hjsonToJson(String hjson) {
+        return JsonValue.readHjson(hjson).toString();
+    }
+
+    /**
+     * Validates json config against json schema.
+     * @param schemaObj : schema object
+     * @param jsonConfig : config to be validated against schema
+     * @return true or false
+     */
+    public boolean isConfigValid(JSONObject schemaObj, String jsonConfig) {
+
+        try {
+            JSONObject data = new JSONObject(new JSONTokener(jsonConfig));
+            Schema schema = SchemaLoader.load(schemaObj);
+            schema.validate(data);
+            return true;
+        } catch (Exception e) {
+            e.printStackTrace();
+            log.error(e.getMessage());
+            return false;
+        }
+    }
+
+    /**
+     * Checks whether input is null or empty.
+     * @param input : input string
+     * @return true or false
+     */
+    public boolean isNullOrEmpty(String input) {
+        return (input == null || input.trim().length() == 0);
+    }
+
+    /**
+     * Loads appropriate schema per input schema type.
+     * @param schemaType i.e. table, security or variable
+     * @return jsonObject of schema
+     * @throws IOException
+     */
+    public JSONObject schemaToJsonObject(String schemaType) throws IOException {
+
+        switch (schemaType) {
+        case SCHEMA_TYPE_TABLE:
+            return loadSchema(ELIDE_TABLE_VALIDATION_SCHEMA);
+
+        case SCHEMA_TYPE_SECURITY:
+            return loadSchema(ELIDE_SECURITY_SCHEMA);
+
+        case SCHEMA_TYPE_VARIABLE:
+            return loadSchema(ELIDE_VARIABLE_SCHEMA);
+        default:
+            return null;
+        }
+    }
+
+    /**
+     * Converts json string to appropriate POJO.
+     * @param inputConfigType ex. table, security or variable
+     * @param jsonConfig input json config
+     * @return config pojo
+     * @throws Exception
+     */
+    public Object getModelPojo(String inputConfigType, String jsonConfig) throws Exception {
```

Comment:
You can use Java Generics here so this class will return the correct type (instead of Object).  Then the calling code won't need to cast.